ReleaseNotes/eng/Altibase_7_3_0_0_1_Release_Notes.md

```diff
@@ -0,0 +1,856 @@
+
+To enhance JDBC fetch performance, ResultSet object are now reused. When multiple ResultSet objects are created from the same PreparedStatement object, the first ResultSet object is reused. If you do not want to reuse ResultSet objects, you can change the value of the reuse_resultset property in the JDBC connection properties to false.
+
+</br>
+
+#### 2.1.13 High Availability
+
+##### Enhanced DDL PVO Stability
+
+Improved reliability by strengthening exception handling in the DDL PVO phase.
+
+##### Improved Protocol Validation
+
+Imporved to prevent abnormal server termination and behavior caused by the transmission of invalid packets (malformed packets). Packet validity is now checked during protocol processing, and in cases of abnormalities, the client's connection is terminated, and diagnostic logs are generated. To enable this feature, the default value of CM_MSGLOG_FLAG has been set to 3, and the default value of SERVER_MSGLOG_FLAG has been changed to 15.
+
+##### Enhanced Transaction Stability - Multiple Rollback Segment
+
+The maximum number of concurrent transactions for disk table that can be executed simultaneously has been expanded from the previous 512 to 16,384
+
+#### Improved Undo Tablespace Reusability
+
+To enhance the stability of Undo tablespace reuse, unnecessary associations between the undo tablespace and disk indexes have been removed, thereby eliminating potential bug-related risks. Additionally default and maximum values of related properties have been changed to improve disk page space efficiency.
+
+- INDEX_INITTRANS maximum value has been increased from 30 to 50.
+- Default and maximum values for INDEX_MAXTRANS have been changed from 30 to 50.
+
+#### 2.1.14 DBeaver Package
+
+Provide a package for DBeaver on Windows.
+
+</br>
+
+### 2.2 Changes
+
+The following describes the features that DBAs and developers need to be aware of, which include additions, modifications, and removals.
+
+#### 2.2.1 Database Version
+
+Version by Database Component
+
+| Altibase server/client Version | Database Binary Version | Meta Version | Communication Protocol Version | Replication Protocol Version |
+| :----------------------------: | :---------------------: | :----------: | :----------------------------: | :--------------------------: |
+|           7.1.0.8.8            |          6.5.1          |    8.11.1    |             7.1.7              |            7.4.7             |
+|           7.3.0.0.1            |          7.3.0          |    9.3.1     |             7.1.8              |            7.4.9             |
+
+#### 2.2.2 Compatibility
+
+##### Database Binary Version
+
+The database binary version indciates the compatibility of the database image file and log file.
+
+The database binary version has been updated due to enhancements in the logging structure of log files. As a result, Altibase 7.3 and earlier versions are not compatible, requiring migration efforts when upgrading to Altibase 7.3.
```

Comment:
사용자가 직접 마이그레이션 할 수가 없을거예요. migration 작업이 꽤 크고, 기술지원으로요청하기때문에 require migration efforts 로 표현했습니다.
logging structure가 수정되었기 때문에,  호환되지 않고, 업그레이드 하려면 migration이 필요하다.는 표현입니다.
Altibase 7.3 and earlier versions are not compatible, so migration is required to upgrade from the earlier version. 는 어떠한지요?

Reply:
The database binary version indicates the compatibility between database image files and log files. The updated database binary version of Altibase 7.3 is not compatible with the lower version anymore because the logging structure of log files is improved. Therefore, if users want to upgrade the existing Altibase version to Altibase 7.3, users should request migration tasks from us(혹은 기술지원팀) in advance.

우선 460번째 줄의 문장을 명확하게 수정했습니다. 뒤의 문장은 설명해주신 내용을 바탕으로 다시 구성해봤는데 한번 확인해주세요~

Reply:
improved -> changed. 가 더 나을것 같습니다.
음,, 마이그레이션을 반드시 기술지원 거쳐서 해야 한다고 명시할필요는 없지 않을까 해요.. (사실은 꼭 거치지만요,,)
그래서 Therefore, if users want to upgrade the existing Altibase version to Altibase 7.3, migration tasks are required.마이그레이션이 필요 하다. 정도로만 작성하는게 좋을 것 같습니다.

Reply:
넵 반영해서 수정했습니다.